### DISCUSSION TITLE
need help about the tracker

### DISCUSSION THREAD

**Discussion by MothDude** [besteon/Ironmon-Tracker#485] [General]
> Error starting up the Tracker: Unable to load the required main Tracker file.
> The "Ironmon-Tracker.lua" script file should be in the same folder as the other Tracker files that came with the release download.
idk how to fix that

**Comment by UTDZac** [comment]
Be sure to follow the install guide on our wiki here on this Github. There is an Easy Install method, which will properly setup the tracker for you. These instructions can also be found on the release page.

https://github.com/besteon/Ironmon-Tracker/releases/latest

Make sure to do each step carefully and thoroughly before moving on to the next step.

In this case, your issue is that you havent properly downloaded the files needed to have the tracker install itself, or you downloaded the fully install and move files around or out of the tracker folders. You should not move the tracker files from their internal folder. Dont put the files inside other application folders like Bizhawk.

**Reply by MothDude** [reply]
![imagem_2024-12-08_170057554](https://github.com/user-attachments/assets/fa805bd2-980e-4dc1-beac-2facd3f3f9ce)
dude,i tried but keep with this

**Reply by MothDude** [reply]
i dont know what i need to do to fix it,i couldnt figure out tbh

**Reply by Fellshadow** [reply]
The issue is probably the accented á character in your PC username, accented characters in the filepath tends to block the tracker from accessing its files

You'll need to move the tracker folder to somewhere that doesn't have any accented characters in any folder name, maybe Downloads could work for you
see here for more info - https://github.com/besteon/Ironmon-Tracker/wiki/FAQ-&-Troubleshooting#folderpermission-issues

**Reply by MothDude** [reply]
thx dude,i notice it was on my desktop paste,thank you sir,happy christmas😁😁
